.github/workflows/build-with-bal-test-native.yml

```diff
@@ -30,7 +30,7 @@ jobs:
     call_stdlib_workflow:
         name: Run StdLib Workflow
         if: ${{ github.event_name != 'schedule' || (github.event_name == 'schedule' && github.repository_owner == 'ballerina-platform') }}
-        uses: ballerina-platform/ballerina-standard-library/.github/workflows/build-with-bal-test-native-template.yml@main
+        uses: ballerina-platform/ballerina-standard-library/.github/workflows/build-with-bal-test-graalvm-template.yml@main
```

Comment:
Need to rename the file: `build-with-bal-test-graalvm.yml`

Reply:
Fixed with [b205270787e95dd2902ee97b4ef25a0475bab757](https://github.com/ballerina-platform/module-ballerina-email/pull/686/commits/b205270787e95dd2902ee97b4ef25a0475bab757)